packages/devtools/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.6 - 2019-08-14
```

Comment:
We should clean up this changelog (we've published `0.1.5` and `0.1.6-dev.2` but not a `0.1.6` stable), and decide on the right package version.

cc @jacob314 @kenzieschmoll